File: ckgame-java/src/main/java/com/ckgame/world/TradeRoute.java
package com.ckgame.world;

/** 贸易路线：起点-终点、贸易量与汇率。 */
public record TradeRoute(String from, String to, double tradeVolume, double exchangeRate) {
    public String key() { return from + "|" + to; }
}

File: ckgame-ts/src/world/TradeRoute.ts
/** 贸易路线：起点-终点、贸易量与汇率。 */
export interface TradeRoute {
  from: string;
  to: string;
  exchangeRate: number;
  tradeVolume: number;
}

export function tradeRouteKey(route: TradeRoute): string {
  return `${route.from}|${route.to}`;
}
